src/definitions.jl

```diff
@@ -271,7 +271,9 @@ summary(p::ScaledPlan) = string(p.scale, " * ", summary(p.p))
 *(p::Plan, I::UniformScaling) = ScaledPlan(p, I.λ)
 
 # Normalization for ifft, given unscaled bfft, is 1/prod(dimensions)
-normalization(::Type{T}, sz, region) where T = one(T) / Int(prod(sz[r] for r in region))::Int
+function normalization(::Type{T}, sz, region) where T
+    one(T) / Int(reduce(*, (sz[r] for r in region); init=1))::Int
```

Comment:
This choice of `init` value makes the assumption that `sz` has element type `Int`, which is the case based on how this function is used but needn't be true in general since there's no type restriction on `sz`.

Unrelated to the current PR but it seems the validity of `region` as a subset of the indices of `sz` is never checked nor ensured. Perhaps worth addressing while this code is being touched?

The following seems to have identical performance to what you've proposed and similarly triggers no errors from JET:
```suggestion
    @boundscheck !isempty(region) && checkindex(Bool, eachindex(sz), region)
    one(T) / mapreduce(r -> Int(@inbounds sz[r]), *, region; init=1)
```

Reply:
> This choice of init value makes the assumption that sz has element type Int, which is the case based on how this function is used but needn't be true in general since there's no type restriction on sz

This is something that I was thinking about, and was considering using something like `oneunit(eltype(sz))` instead of `1`. Perhaps this would be better, if we're disallowing the 0d case?

Reply:
The bounds checking leads to a test failure downstream because region is a `Tuple{Int}`